examples/swagger/contract.yml

```diff
@@ -0,0 +1,105 @@
+swagger: "2.0"
```

Comment:
Er bestond al ee openapi spec in ./examples/webserver/openapi.yaml. Is dit een kopie?